src/httpserver.cpp

```diff
@@ -71,8 +71,7 @@ class WorkQueue
     /** Mutex protects entire object */
     CWaitableCriticalSection cs;
     CConditionVariable cond;
-    /* XXX in C++11 we can use std::unique_ptr here and avoid manual cleanup */
-    std::deque<WorkItem*> queue;
+    std::deque<std::unique_ptr<WorkItem> > queue;
```

Comment:
@paveljanik because `>>` is an operator. But yes with c++11 this is apparently parsed differently so will remove the space.